brainstormforce/ultimate-addons-for-gutenberg
# Fix: Table of content - Title alignment.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/40d652a7-be07-469a-88fa-1c85fa89f40e?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a TOC in Master with Title at the center.
- Switch to next-release you will see title alignment set to left.
- Switch to above branch which fixes this issue.

### Checklist:
